LanguageFeatures/Augmentations/augmenting_constructors_A09_t03.dart

```diff
@@ -2,29 +2,54 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @assertion At a high level, a non-redirecting generative constructor marked
-/// `augment` may:
-/// - Augment the constructor with an additional constructor body (bodies are
-///   invoked in augmentation order, starting at the introductory declaration).
+/// @assertion It's a compile-time error if there is no matching declaration.
 ///
 /// @description Checks that it is a compile-time error to augment a default
 /// unnamed constructor (that doesn't exist during augmentation).
```

Comment:
This comment is _very_ useful, especially when the reader may not remember exactly why this is an error!